lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}

```

Comment:
I like that last idea better. Not a fan of the idea of global subscriptions hanging off Cable being the default api because to me, if I saw that in code, I'd think, "oh hey, there's an undocumented appearance class in ActionCable, let me go dig through the gem to find it."

And to further your point that code should be rewritten as:

```coffee
cable = Cable.createConsumer "xs://localhost:3000/app_cable"

appearance = cable.subscriptions.create "AppearanceChannel",
  appear: ->
    @perform 'appear', ...

$(document).on 'page:change', ->
  appearance.appear()
```

So should we create a cable.coffee instead of index.coffee and add channel subscriptions that are generated to the end of cable.coffee (if it exists) instead?